go-controller/pkg/ovn/logical_switch_manager/logical_switch_manager.go

```diff
@@ -16,6 +17,9 @@ import (
 	utilnet "k8s.io/utils/net"
 )
 
+// ErrNotFound is used to inform the object or table was not found in the cache
+var ErrNotFound = errors.New("object not found")
```

Comment:
nit: it looks like the error message is a bit redundant.

could this be more specific?
```
var SwitchNotFound = errors.New("switch not found")
```
and then the errors below could be less specific
```
return fmt.Errorf("unable to allocate ips: %w", switchName, SwitchNotFound)
...
return fmt.Errorf("unable to allocate hybrid overlay: %w", switchName, SwitchNotFound)
...
return fmt.Errorf("unable to release ips: %w", switchName, SwitchNotFound)
```

Reply:
ack. Will do, @jcaamano !